tests/conftest.py

```diff
@@ -93,5 +98,29 @@ def safe_data_file(chain, safe_contract):
 
 @pytest.fixture
 def safe(safe_data_file):
-    # TODO: Mock `SafeAccount.client` or use local client
     return SafeAccount(account_file_path=safe_data_file)
+
+
+@pytest.fixture
+def token(deployer):
+    contract = ContractType.parse_file(contracts_directory / "Token.json")
+    return deployer.deploy(ContractContainer(contract))
+
+
+@pytest.fixture
+def vault(deployer, token):
+    vault = ContractContainer(ContractType.parse_file(contracts_directory / "VyperVault.json"))
+    return deployer.deploy(vault, token)
+
+
+@pytest.fixture
+def foundry(networks):
+    with networks.ethereum.local.use_provider("foundry") as provider:
+        yield provider
+
+
+@pytest.fixture
+def multisend():
+    ms = MultiSend()
+    ms.inject()
```

Comment:
think this is a classmethod

Reply:
It's valid to call classmethods with instances. We need the instance anyway.
I can change to calling inject with the class and still returning the instance though.

As an aside, if you want to avoid allowing instances to call the method entirely, you can make it a `@staticmethod`.. I never really do that though, I don't really see the point.